### ISSUE TITLE
Warning: Illegal characters in latest commits

### ISSUE THREAD

**Issue by MajorCooke** [ZDoom/gzdoom#2340]
This is AFTER [this commit](https://github.com/ZDoom/gzdoom/commit/ec227383eddb3438d2d295cce79ec1ff29e54b80)

```
12>\src\common\thirdparty\math\fastsin.cpp(1,1): warning C4828: The file contains a character starting at offset 0x8dd that is illegal in the current source character set (codepage 65001).
12>\src\common\textures\hires\xbr\xbrz.cpp(1,1): warning C4828: The file contains a character starting at offset 0x3fad that is illegal in the current source character set (codepage 65001).
3>\src\common\rendering\hwrenderer\data\hw_skydome.cpp(1,1): warning C4828: The file contains a character starting at offset 0xc24 that is illegal in the current source character set (codepage 65001).
12>\src\playsim\p_actionfunctions.cpp(3244,1): warning C4828: The file contains a character starting at offset 0x16a8b that is illegal in the current source character set (codepage 65001).
12>\src\common\textures\bitmap.cpp(1,1): warning C4828: The file contains a character starting at offset 0x1e63 that is illegal in the current source character set (codepage 65001). 
(THIS ONE REPEATED 5 TIMES.)
```

**Comment by coelckers** [comment]
I know, In that commit I activated MSVC's UTF-8 handling and apparently a few files out there are still having some legacy extended characters.

These messages are not really helpful because they do not give me the actual line the error is on which makes it hard to locate them.

**Comment by coelckers** [comment]
fixed in e27cbe5bd938093eed11ddd345fc1425a41f7477
